test/keyword_arguments.jl

```diff
@@ -401,4 +401,28 @@ end
     @test df2 == correct
 end
 
+@testset "Multiple arguments #399" begin
+    correct = df[df.a .== 1, :]
+    correct_view = view(df, df.a .== 1, :)
+
+    df2 = @subset(df, :a .== 1, :b .== 3; view = true)
+    @test df2 ≈ correct_view
```

Comment:
Can you additionally check type of `df2`?

Reply:
Sorry, I missed this!